weather_underground_base.py

```diff
@@ -40,59 +41,37 @@ class WeatherUndergroundBase(RESTPolling):
     def __init__(self):
         super().__init__()
         self._api_endpoint = 'conditions'
-        self._response_key = 'current_observation'
 
     def configure(self, context):
         super().configure(context)
 
-    def _process_response(self, resp):
-        """ Extract weather conditions from response.
+    def process_signals(self, signals):
+        response = None
+        weather_signals = []
 
-        Args:
-            resp (Response)
+        for signal in signals:
+            for locations in self.queries():
```

Comment:
I don't think we need (nor should) `queries` to be a list anymore. Maybe just make a `city` and `state` prop and use those.